VMBackup/main/guestsnapshotter.py

```diff
@@ -338,6 +413,9 @@ def snapshotall(self, paras, freezer, g_fsfreeze_on):
             if exceptOccurred and thaw_done == False and all_snapshots_failed:
                 self.logger.log("Trying sequential snapshotting as parallel snapshotting failed")
                 snapshot_result, blob_snapshot_info_array, all_failed, exceptOccurred, is_inconsistent,thaw_done, unable_to_sleep, all_snapshots_failed =  self.snapshotall_seq(paras, freezer, thaw_done, g_fsfreeze_on)
+        
+        self.logger.log("Adding to the telemetry " + json.dumps(blobMetadataTelemetryMessage))
+        HandlerUtil.HandlerUtility.add_to_telemetery_data("BlobMetadataMessage", json.dumps(blobMetadataTelemetryMessage))
```

Comment:
make telemetry similar to Windows extension (only count if count > 0)

Reply:
Done